projects/penpie/index.js

```diff
@@ -12,9 +12,12 @@ async function tvl(api) {
   });
   const poolInfos = await api.multiCall({ abi: 'function getPoolInfo(address) view returns ( uint256 emission, uint256 allocpoint, uint256 sizeOfPool, uint256 totalPoint)', calls: poolTokens, target: masterPenpie, })
   const symbols = await api.multiCall({ abi: 'erc20:symbol', calls: poolTokens, })
+  
   poolTokens.forEach((token, i) => {
-    if (symbols[i] === 'vlPenpie' || symbols[i] === 'mPendle' || symbols[i] === 'mPendleOFT' || symbols[i] === 'mPendleSV') {
-      token = PENDLE
+ 
+    if (symbols[i] === 'vlPenpie' ){
```

Comment:
isnt this already counted towards staking?

Reply:
mpendle,pendle,penpie prices are now fetchable directly. hence removed that line and added an if condition only for vlpenpie which is locked version of penpie

Reply:
yeah, but we shouldnt count staked penpie as part of tvl, as it is already counted as part of staking?